### PR TITLE
replace sourcefource ticket management with github Issue management

### PR DESCRIPTION
**Pull request by smita786**
Signed-off-by: Smita Kumari <kumarismita62@gmail.com>

### CODE REVIEW

**Comment by smita786** [comment]
Hey @nijel , any idea why migration is failing during travis build? Its throwing error  "The "date" option does not exist. ".  
I didn't change anything in Migration part. and it's working fine on my system.

**Comment by nijel** [comment]
Seems to be described here: https://github.com/cakephp/migrations/issues/117 . I've updated cakephp in git and it seems to have fixed it.

File: src/Controller/DevelopersController.php [hunk 1]
```diff
@@ -36,12 +38,13 @@ class DevelopersController extends AppController {
...
 
 	public function beforeFilter(Event $event) {
+        parent::beforeFilter($event);
 		$this->GithubApi->githubConfig = Configure::read('GithubConfig');
 		$this->GithubApi->githubRepo = Configure::read('GithubRepoPath');
 	}
 
 	public function login() {
-		$url = $this->GithubApi->getRedirectUrl('user:email');
+		$url = $this->GithubApi->getRedirectUrl('user:email,public_repo,repo');
```
Comment (nijel) [inline src/Controller/DevelopersController.php]: The repo permission is not needed, I've removed it in 8ad4e63949ee332ea78cec5d44f58221caf9b06c/

**Comment by smita786** [comment]
Thanks Michal :-)

**Pull request merged**